packages/accessibility/.eslintignore

```diff
@@ -0,0 +1,4 @@
+coverage/
```

Comment:
Does eslint not fall back to using gitignore?

Reply:
You can explicitly use `.gitignore` but there is no auto fallback: https://github.com/eslint/eslint/issues/5848#issuecomment-210212138

Reply:
That's disappointing... I wonder if in the future we should set it up to use gitignore instead of the separate file to avoid it getting out of sync. (might not work right now since lib is not globally gitignored yet)